### PR TITLE
Add delay for ssh

### PR DESCRIPTION
**Pull request by jesusbv**

### CODE REVIEW

**Review by pdostal — CHANGES_REQUESTED** [review]
This in my opinion does not make any sense as there is already `wait_for_ssh` subroutine.

**Comment by asmorodskyi** [comment]
> This in my opinion does not make any sense as there is already `wait_for_ssh` subroutine.

exactly . and if you have timeouts you need just increase waiting time in `wait_for_ssh`

**Comment by jesusbv** [comment]
> > This in my opinion does not make any sense as there is already `wait_for_ssh` subroutine.
> 
> exactly . and if you have timeouts you need just increase waiting time in `wait_for_ssh`

Ack, feel free to close the PR
I was told it would be a good improvement for the ssh softreboot

File: lib/publiccloud/instance.pm [hunk 1]
```diff
@@ -287,6 +288,7 @@ sub softreboot
     sleep 60;    # wait for the +1 in the previous command
     my $start_time = time();
 
+    sleep($args{delay});
```
Comment (grisu48) [inline lib/publiccloud/instance.pm:291]: This is difficult. This routine is also used for measuring of boottime (see the line before), so adding an additional delay here might interfere with the boottime test. We could update line 288 though and state, that if delay is set, this subroutine might not return the accurate boottime.

**Pull request closed without merging**